web/src/search_suggestion.ts

```diff
@@ -508,7 +508,7 @@ function get_topic_suggestions(last: NarrowTerm, terms: NarrowTerm[]): Suggestio
 
     return topics.map((topic) => {
         const topic_term = {operator: "topic", operand: topic, negated};
-        const terms = [...suggest_terms, topic_term];
```

Comment:
It seems like we can completely remove `suggest_terms` from this function then?

Reply:
That would also make the second commit more readable, I was confused for a bit that the channel might still be part of the suggestion.